gensrc/thrift/FrontendService.thrift

```diff
@@ -694,15 +694,15 @@ struct TReplicaMeta {
     2: optional i64 backend_id
     3: optional i32 schema_hash
     4: optional i64 version
-    5: optional i64 version_hash
+    5: optional i64 version_hash // Deprecated
```

Comment:
These `Deprecated` comments did not make sense in practice, because no deprecated annotation will be generated in Java or C++ code.

Reply:
If they are truly unused anymore, please comment them out

Reply:
Because FE code paths is not removed, so it's can not be removed.
Thrift have no [deprecated=true] grammer.

Reply:
If FE still depends on these fields, maybe they should not be marked as deprecated

Reply:
FE will remove the PR in the following pull request. The deprecated indicator is to give a hint to not use the variable any more.